### PR TITLE
Convert plugin to new OMF architecture

### PR DESCRIPTION
**Pull request by bpinto**

### CODE REVIEW

File: completions/git.fish [hunk 1]
```diff
@@ -1,44 +1,3 @@
...
-#
-#  1. Install this file in your `~/.config/fish/completions` folder.
-#
-#
-# The Fine Print
-# --------------
-#
-# Copyright (c) 2012-2015 [Justin Hileman](http://justinhileman.com)
-#
-# Distributed under the [MIT License](http://creativecommons.org/licenses/MIT/)
```
Comment (bobthecow) [inline completions/git.fish:28]: leave the header.

this file is literally copied from another repo. don't mess things up by removing copyright headers :)

Comment (bpinto) [inline completions/git.fish:28]: Re-added.

File: init.fish [hunk 2]
```diff
@@ -0,0 +1,12 @@
+function init -a path --on-event init_plugin-git-flow
```
Comment (bobthecow) [inline init.fish]: Isn't this just `init_git-flow` not `init_plugin*`?

Comment (bpinto) [inline init.fish]: It is, I created the template with the wrong name... this was a leftover!

**Comment by bpinto** [comment]
Could you confirm it's working? I don't use `git-flow` myself.

**Comment by bobthecow** [comment]
looks good. :+1:

**Pull request merged**